hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java

```diff
@@ -901,13 +923,20 @@ else if (ro.type().isGrantOrUser()) {
             result = null;
         }
 
-        // log user action
-        if (!(actionPerm == HugePermission.READ && ro.type().isSchema())) {
+        // log user action, limit rate for each user
+        Id usernameId = IdGenerator.of(username);
+        RateLimiter auditPerUserRate = this.auditPerUserRates.get(usernameId);
+        if (auditPerUserRate == null) {
+            auditPerUserRate = RateLimiter.create(this.logRate);
+            this.auditPerUserRates.update(usernameId, auditPerUserRate);
+        }
+        if (auditPerUserRate.tryAcquire() &&
```

Comment:
move tryAcquire behind of condition `!(actionPerm == HugePermission.READ && ro.type().isSchema())`